randovania/games/dread/logic_database/Artaria.txt

```diff
@@ -1581,7 +1581,6 @@ Extra - asset_id: collision_camera_015
       Any of the following:
           Space Jump
           Climb Sloped Surfaces (Beginner) and Use Spin Boost
-          Speed Booster and Speed Booster Upgrade and Speed Booster Conservation (Beginner) and Disabled Door Lock Randomizer
```

Comment:
This could be re-added but with adding Spin Boost, and then putting Speed Booster Upgrade and Disabled Door Lock Randomizer into an Or clause instead.